docs/features/index.rst

```diff
@@ -17,6 +17,10 @@
 Features
 ========
 
+A **Feature** is the highest-level logical entity, representing a set of interrelated components in respect of the S-CORE software platform. It is the primary unit of the management of these components and contains the belonging feature requirements and feature architecture for them. For further explanation see the `Building blocks concept <https://eclipse-score.github.io/process_description/main/general_concepts/score_building_blocks_concept.html>`_.
```

Comment:
Not sure how links should be given here. This points to an absolute URL that is only meaningful after "compilation" and publishing at this site - so it would not work if someone "compiles" and deploys locally / on any other system / at any other site. Might well be the correct way to do this, just noticed it.
Applies also to the other links added in this PR.